packages/frontend/component/src/components/member-components/member-limit-modal.tsx

```diff
@@ -0,0 +1,49 @@
+import { ConfirmModal } from '@affine/component/ui/modal';
+import { useAFFiNEI18N } from '@affine/i18n/hooks';
+import { useCallback } from 'react';
+
+export interface MemberLimitModalProps {
+  isFreePlan: boolean;
+  open: boolean;
+  setOpen: (value: boolean) => void;
+  onConfirm: () => void;
+}
+
+export const MemberLimitModal = ({
+  isFreePlan,
+  open,
+  setOpen,
+  onConfirm,
+}: MemberLimitModalProps) => {
+  const t = useAFFiNEI18N();
+  const handleConfirm = useCallback(() => {
+    setOpen(false);
+    if (isFreePlan) {
+      onConfirm();
+    }
+  }, [onConfirm, setOpen, isFreePlan]);
+
+  return (
+    <ConfirmModal
+      open={open}
+      onOpenChange={setOpen}
+      title={t['com.affine.payment.member-limit.title']()}
+      description={t[
+        isFreePlan
+          ? 'com.affine.payment.member-limit.free.description'
+          : 'com.affine.payment.member-limit.pro.description'
+      ]()}
```

Comment:
"Each ${plan} user can invite up to ${quota} members to join their workspace." 
the plan and quota can query by useUserQuota(), split this sentence as independent i18n entry